static/js/main.js

```diff
@@ -241,33 +242,49 @@ function addScreenshot(screenshot) {
                             value='${time_input_value}' />
                 </div>
                 <div>
-                    <button onclick='flipImage("${screenshot.id}")'>Flip Image</button>
+                    <span>
+                        <button onclick='flipScreenshot("${screenshot.id}", true)'>Vertical Flip</button>
+                        <button onclick='flipScreenshot("${screenshot.id}", false)'>Horizontal Flip</button>
+                    </span>
                 </div>
+                <div>
+                    <button onclick='deleteScreenshot("${screenshot.id}")'>Delete</button>
+                </div> 
             </div>
         </div>
     `;
 
     document.querySelector('.screenshot-area').insertAdjacentHTML('beforeend', html);
 }
 
-// Flip an image in the screenshot area by adding it the the canvas flipped horizontally.
-function flipImage(screenshot_id) {
-    // Find correct image, add to canvas to flip.
+// Flip a screenshot by adding it the the canvas flipped horizontally or vertically.
+function flipScreenshot(screenshot_id, flip_vertical) {
     let img = document.querySelector(`.screenshot[data-id="${screenshot_id}"] img`);
 
+    // Default values for horizontal flip;
+    let direction = 'flipped_horizontally';
+    let scale_x = -1;
+    let scale_y = 1;
+    let draw_x = -img.naturalWidth;
+    let draw_y = 0;
+
+    if (flip_vertical) {
+        direction = 'flipped_vertically';
+        scale_x = 1, scale_y = -1;
+        draw_x = 0, draw_y = -img.naturalHeight;
```

Comment:
In our style (most styles I run into), we don't want to set multiple variables on the same line.
(Sometimes this is relaxed for declarations (e.g. `let a = b = 1;`), but not for already defined variables. I never relax this for declarations as I think it makes it easy to miss things (and some languages like C++ can have tricky comma operators)).